drivers/SmartThings/zigbee-lock/fingerprints.yml

```diff
@@ -124,6 +124,17 @@ zigbeeManufacturer:
     manufacturer:
     model: E261-KR0B0Z0-HA
     deviceProfileName: lock-battery
+  # TOTEM
+  - id: TOTEM/H60/H90
+    deviceLabel: TOTEM Door Lock
+    manufacturer: TOTEM
+    model: H60/H90
+    deviceProfileName: lock-battery
```

Comment:
Does this device support the lockCodes capability ? If so  - the profile should be updated.

Reply:
@FrankSpringfield This comment will need a response or on a related ticket before we can proceed. Thank you.

Reply:
@lelandblue 
I'm sorry for responding too late. I have a day off yesterday.

As for the answer:
Yep, this device does support the lockCodes capability.
I'll change the deviceProfileName from lock-battery to base-lock and test whether it works well.